packages/select/test/listItemsProps.ts

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
```

Comment:
1. ~~2018~~ 2019 😂 
2. `listItemsPropsTests.ts`
3. these tests are 🔥!! thank you 🙏

Reply:
I copied the Copyright comment from another file in this folder and didn't even look at the year. Should I updated other files in this folder, or is this year supposed to mark the creation or last update of the file?

Reply:
nah just new files